### PR TITLE
Update LDAP Docker image

### PR DESCRIPTION
**Pull request by tkrille**
The old one isn't available anymore.

### CODE REVIEW

**Comment by tkrille** [comment]
I don't know why CircleCI does not find any tests. Just merge it, please.

File: src/main/docker/ldap/Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM osixia/openldap:1.0.9
+FROM osixia/openldap:1.1.7
```
Comment (dacrome) [inline src/main/docker/ldap/Dockerfile]: What about latest?

Comment (tkrille) [inline src/main/docker/ldap/Dockerfile]: Alright

Comment (tkrille) [inline src/main/docker/ldap/Dockerfile]: Done

**Pull request merged**